e2e/nomostest/nt.go

```diff
@@ -148,23 +147,20 @@ type NT struct {
 	// caCertPath is the path to the CA cert used for communicating with the Git server.
 	caCertPath string
 
-	// gitRepoPort is the local port that forwards to the git repo deployment.
-	gitRepoPort int
+	// gitRepoPortForward is the local port forwarding for the git server deployment.
+	gitRepoPortForward *portforwarder.PortForwarder
 
-	// gitRepoPodName is the pod name of the git repo.
-	gitRepoPodName string
+	// prometheusPortForward is the local port forwarding for the prometheus deployment.
+	prometheusPortForward *portforwarder.PortForwarder
```

Comment:
nit: these could be named `prometheusProxy` and `gitRepoProxy`. Ro at least add -er to match the structs.

Reply:
done